src/kirby/components/badge/badge.component.tns.html

```diff
@@ -0,0 +1,4 @@
+<AbsoluteLayout class="kirby-badge">
```

Comment:
We are already scoped, so maybe just:
```suggestion
<AbsoluteLayout class="badge">
```
OR simply:
```suggestion
<AbsoluteLayout class="wrapper">
```

Reply:
Its been changed to .badge and the innerlabel is now .badge-label